lib/nexmo_developer/app/views/contribute/tutorials-and-use-cases/use-cases.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Creating a Use Case
+navigation_weight: 2
+---
+
+# Creating a Use Case
+
+A Use Case shows how readers how they might implement a real-world application.
+
+Use Cases are simpler to write than Tutorials, because the entire content exists in a single markdown file.
+
+> **Wait!** Are you sure that a Use Case is the right format for what you want to achieve? Please read our [Tutorials and Use Cases guide](/contribute/guides/tutorials-and-use-cases) before continuing.
+
+Create the file for your Use Case in the appropriate language folder for your locale in the `_use_cases` directory, e.g. `_use_cases/en`, for English.
+
+Begin your Use Case with a Level 1 heading (`#`) and suitable introductory paragraph.
+
+Each main topic within your Use Case should be a Level 2 heading (`##`). You can create subheadings under each of the main topics, but please try to not go beyond Level 3 (`###`) unless absolutely necessary. Please avoid anything beyond a Level 4 (`####`) heading.
+
+![Creating a Use Case](/assets/images/contributing/create-a-uc.png)
```

Comment:
This image is not in the directory

Reply:
@bencgreenberg Thanks - the images directory is in `.gitignore` ... is there a reason for that?

Reply:
I believe the intention is to strip Station of specific portal content and move most of that to instances of Station (VBC, ADP, etc.)